src/app/components/pages/quizzes/MyQuizzes.tsx

```diff
@@ -150,6 +151,11 @@ const MyQuizzesPageComponent = ({user}: MyQuizzesPageProps) => {
         }
     };
 
```

Comment:
I like the comment on the original SetQuizzes file that this logic is from to explain:

```suggestion

    // If the user is event admin or above, and the quiz is hidden from teachers, then show that
    // Otherwise, show if the quiz is visible to students
```

Otherwise, everything looks good